README.md

```diff
@@ -24,46 +24,56 @@ Kubernetes Cluster with Knative, if you dont have an OpenShift cluster see [try.
   </thead>
 
   <tr>
-    <td>service_name</td>
-    <td>Yes</td>
-    <td>
-      The Knative Service Name.
+    <td>container_image</td>
+    <td>No</td>
+    <td>The container image to use for service. Not required if <code>service_operation</code> is set to <code>delete</code>. </td>
+  </tr>
+
+  <tr>
+    <td>force_create</td>
+    <td>No</td>
+    <td>Pass `--force` to `kn create` to replace the existing service, instead of failing if the service already exists.
+    Defaults to <code>false</code>.
     </td>
   </tr>
 
   <tr>
-    <td>service_namespace</td>
-     <td>No</td>
-    <td>The Kubernetes Namespace to deploy to. Defaults to current context's namespace. </td>
+    <td>registry_password</td>
+    <td>No</td>
+    <td>The registry user password or token. Required if image registry is private. </td>
   </tr>
 
   <tr>
-    <td>service_operation</td>
+    <td>registry_user</td>
     <td>No</td>
-    <td>The `kn` service operation <code>create</code>, <code>update</code>, <code>apply</code> etc. 
-    Defaults to <code>create</code></td>
+    <td>The registry user to use to create image pull secret. Required if image registry is private. </td>
   </tr>
 
   <tr>
-    <td>container_image</td>
+    <td>service_name</td>
     <td>Yes</td>
-    <td>The container image to use for service. </td>
+    <td>
+      The Knative Service name.
+    </td>
   </tr>
 
   <tr>
-    <td>service_params</td>
-    <td>No</td>
-    <td>The extra service parameters to pass to the service. </td>
+    <td>service_namespace</td>
+     <td>No</td>
+    <td>The Kubernetes Namespace to deploy to. Defaults to current context's namespace. </td>
   </tr>
+
   <tr>
-    <td>registry_user</td>
+    <td>service_operation</td>
     <td>No</td>
-    <td>The registry user to use to create image pull secret. Required if image registry is private. </td>
+    <td>The `kn` service operation <code>create</code>, <code>update</code>, <code>apply</code> <code>delete</code>. 
+    Defaults to <code>create</code>.</td>
   </tr>
+
   <tr>
-    <td>registry_password</td>
+    <td>service_params</td>
     <td>No</td>
-    <td>The registry user password or token. Required if image registry is private. </td>
+    <td>The extra service parameters to pass to the service. </td>
```

Comment:
These are passed to the `kn` command right? That's not really clear from this.

How about "Any extra parameters to append to the `kn <operation>` call`.

I also prefer "arguments" to "parameters" for command-line calls, but that's subjective.